.github/deployments/helmfile.yaml

```
@@ -0,0 +1,21 @@
+# helmfile for Rasa X deployment
+repositories:
+- name: rasa-x
+  url: https://rasahq.github.io/rasa-x-helm
+
+releases:
+  - name: example
+    namespace: default
+    chart: rasa-x/rasa-x
+    values:
+      - nginx:
+          enabled: false
+      - ingress:
+          annotations:
+            kubernetes.io/ingress.class: nginx
+            nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+            nginx.ingress.kubernetes.io/enable-cors: "true"
+          hosts:
+            - host: example.test.com
+              paths:
+                - /
```

```suggestion
      - rasax:
          disableTelemetry: true
          initialUser:
            password: {{ requiredEnv "INITIAL_USER_PASSWORD" }}
      - duckling:
          enabled: false
      - nginx:
          enabled: false
      - ingress:
          annotations:
            kubernetes.io/ingress.class: nginx
            nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
            nginx.ingress.kubernetes.io/enable-cors: "true"
          hosts:
            - host: example.test.com
              paths:
                - /
```
You need a few options more